src/maicos/lib/math.py

```diff
@@ -761,12 +771,13 @@ def transform_sphere(positions: np.ndarray, origin: np.ndarray) -> np.ndarray:
     return trans_positions
 
 
-def compute_rdf_structure_factor(
+def rdf_structure_factor(
     rdf: np.ndarray, r: np.ndarray, density: float
 ) -> tuple[np.ndarray, np.ndarray]:
     r"""Computes the structure factor based on the radial distribution function (RDF).
 
-    The structure factor :math:`S(q)` based on the RDF :math:`g(r)` is given by
+    The structure factor :math:`S(q)` based on an equally spaced RDF :math:`g(r)` is
```

Comment:
Speaking of equally spaced RDFs and then showing a continous integral in the formula is a bit confusing... Is there a reason only equally spaced RDFs are allowed, other than the way its implemented?

Reply:
It is only an implementation limitation. Is just want to add an information that we assume the data is equally spaced. We could also

(a) adjust the code (should be be not too hard)
(b) throw an error if the data is not equally spaced.

Reply:
I think option (b) is fine for now, option (a) would need additional tests, and RDF are nearly always equally spaced